### DISCUSSION TITLE
next static page chunk 404

### DISCUSSION THREAD

**Discussion by tuvshin25** [vercel/next.js#40244] [Help]
I'm running a Next.js on Netlify. We getting 404 for only three pages

GET https://www.gobicashmere.com/de/_next/static/WDyb0rywF_Q6VtUs9PF-F/_buildManifest.js net::ERR_ABORTED 404
GET https://www.gobicashmere.com/de/_next/static/WDyb0rywF_Q6VtUs9PF-F/_ssgManifest.js net::ERR_ABORTED 404
GET https://www.gobicashmere.com/de/_next/static/WDyb0rywF_Q6VtUs9PF-F/_middlewareManifest.js net::ERR_ABORTED 404
GET https://www.gobicashmere.com/de/_next/static/chunks/webpack-63fd9fe6fcb88c16.js net::ERR_ABORTED 404
GET https://www.gobicashmere.com/de/_next/static/chunks/pages/collections/%5Bhandle%5D-744491e2cd7955f8.js net::ERR_ABORTED 404

Our next jS config is 

/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  images: {
    domains: [
      'storage.googleapis.com',
      'chart.googleapis.com',
      'cdn.shopify.com',
    ],
    formats: ['image/avif', 'image/webp'],
    minimumCacheTTL: 720,
    deviceSizes: [640, 750, 828, 1080, 1440, 1920, 2048, 3840],
  },
}

module.exports = nextConfig

Does anyone know what I can do to fix this?

**Comment by icyJoseph** [comment]
Where did you see these errors? Those are Next.js internals which change when you deploy the application. I see that in the currently deployed site all of these load normally. 

Are you seeing this in a logs aggregate? Or perhaps you've fixed the problem? I don't see it anymore.

**Comment by tuvshin25** [comment]
We can see it current live website.
https://www.gobicashmere.com/de/collections/sale
https://www.gobicashmere.com/de/collections/sale-damen
https://www.gobicashmere.com/de/collections/sale-herren

Only this 3 pages.

If we use VPN it's works but from Hong Kong and Mongolia now working.

**Reply by icyJoseph** [reply]
Mmmm yeah from my timezone I see no problem at all, all pages load correctly, no errors in console, nor in the network tab.

**Reply by SilentCC** [reply]
I got same issue here. It happens randomly.

![image](https://github.com/user-attachments/assets/384f2d6b-aa9d-45a0-8bb2-47e7735f6512)

**Reply by dennisjnnh** [reply]
> I got same issue here. It happens randomly.
> 
> ![image](https://private-user-images.githubusercontent.com/26513670/416948643-384f2d6b-aa9d-45a0-8bb2-47e7735f6512.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU1MjM4MjUsIm5iZiI6MTc0NTUyMzUyNSwicGF0aCI6Ii8yNjUxMzY3MC80MTY5NDg2NDMtMzg0ZjJkNmItYWE5ZC00NWEwLThiYjItNDdlNzczNWY2NTEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDI0VDE5Mzg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY2MTQ5NDA3MmMwYmE2ZGZjM2ZiM2QzMGUzMzFjYzc5NzM0YTFlNTM1NThiNDk2MjAwZjBmM2I5ODRiYjIxYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tTIdIBDQxKfNncKO16jkbKa5gIxOYuWOFDU6kX_A7SE)

I notice this behaviour randomly on one page too, did you dig any further into this?
